nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -446,27 +478,8 @@ define(
                 };
             }
 
-            function loadWidget(name) {
-                return new Promise((resolve, reject) => {
-                    require('./tabs/' + name, (Widget) => {
-                        resolve(Widget);
-                    }, (err) => {
-                        reject(err);
-                    });
-                });
-            }
-
             function startTab(tabId) {
                 const selectedTab = controlBarTabs.tabs[tabId];
-                if (selectedTab.widgetModule) {
```

Comment:
this data structure never exists, and the code in the `then` segment causes an error ==> delete it!